src/Kubernetes.WebSocket.cs

```diff
@@ -226,18 +226,20 @@ public Task<WebSocket> WebSocketNamespacedPodAttachAsync(string name, string @na
 
 #if NETCOREAPP2_1
             if (this.CaCert != null)
+            {
                 webSocketBuilder.ExpectServerCertificate(this.CaCert);
-            else
```

Comment:
Aren't these mutually exclusive? I suppose throwing an exception would have been clearer when I first wrote this...

Reply:
Maybe? I sort of lean on "the customer is always right" or "the caller is always right"

If they ship us a config with a ca Cert and noVerifyTLS who are we to judge? We should obey their wishes.

Reply:
Yeah fair enough, that reasoning makes sense to me :)

My personal approach to API design is "forgiving, but also willing to point out when the caller may be making a mistake". If someone specifies both `CaCert` and `SkipTlsVerify`, which one did they really intend to specify? Is it intuitive to them which one we'd select when there's a conflict like this, and do we document this assumption anywhere?

**But** in this case, correct behaviour is dictated by the K8s client config spec which doesn't prevent them specifying both (so I agree with you that there's not much point in stopping them from doing so).